__tests__/app.test.js

```diff
@@ -72,17 +72,17 @@ describe("nc-news", () => {
         .get("/api/articles/1000")
         .expect(404)
         .then(({ body }) => {
-          expect(body.msg).toBe("id not found");
+          expect(body.msg).toBe("article at id not found");
         });
     });
 
     test("GET 400: returns error message when invalid id is given", () => {
-        return request(app)
-          .get("/api/articles/pie")
-          .expect(400)
-          .then(({ body }) => {
-            expect(body.msg).toBe("Something wrong with input");
-          });
-      });
+      return request(app)
+        .get("/api/articles/pie")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Something wrong with input");
+        });
+    });
```

Comment:
Nice to see you're thinking about a custom error handler for incorrect endpoints!